index.js

```diff
@@ -32,12 +32,8 @@ module.exports = customService(async function index(service) {
     useUnifiedTopology: false,
   })
 
-  service.addRawCustomPlugin('GET', '/greetings/:userId', async function handler(req, reply) {
-    const currentUserId = req.params.userId
-    if (!currentUserId) {
-      throw new Error('This API should be consumed only by authenticated users!')
-    }
-    const cursor = this.mongo.db.collection('mycollection').find({ from: currentUserId })
+  service.addRawCustomPlugin('GET', '/greetings', async function handler(req, reply) {
+    const cursor = this.mongo.db.collection('mycollection').find({ from: req.query.from })
```

Comment:
Please add the schema request schema (as it's done in the POST request) with the query parameter `from`. 

Also add a check against falsy values before using it or mark it as required in the request schema.

Reply:
Done!